apps/system/js/utility_tray.js

```diff
@@ -251,6 +282,7 @@ window.UtilityTray = {
           return;
         }
 
+        this.lastTime = evt.timeStamp - this.lastTime;
```

Comment:
nit: it seems weird that `this.lastTime` starts life as a timestamp, but then becomes a time delta. I think better would be to call it `this.touchStartTime` or something above, and then in the `onTouchEnd` function itself we could calculate the delta time as a local var. Like the above comment about passing in the event timeStamp to `onTouchStart` we would have to do the same thing here with the event timeStamp to `onTouchEnd`.